packages/yarnpkg-cli/sources/cli.ts

```diff
@@ -6,7 +6,11 @@ import {YarnVersion}            from '@yarnpkg/core';
 import {main}                   from './main';
 import {getPluginConfiguration} from './tools/getPluginConfiguration';
 
-main({
-  binaryVersion: YarnVersion || `<unknown>`,
-  pluginConfiguration: getPluginConfiguration(),
-});
+export {getDynamicLibs} from "./tools/getDynamicLibs";
+
+if (process.argv[1]) {
```

Comment:
This check doesn't work (for example it would call `main` is the api was used in a script that takes arguments). A better check (but that needs to be adapted to get past the bundler) is this:

```suggestion
if (require.main === module) {
```